helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -621,6 +621,11 @@ logsCollection:
     # Set to 0 to remove any size limit.
     maxRecombineLogSize: 1048576
 
+    # Whether to preserve leading whitespaces in logs.
+    preserveLeadingWhitespaces: false
+    # Whether to preserve trailing whitespaces in logs.
+    preserveTrailingWhitespaces: false
```

Comment:
Why do we need extra parameters when these are already available with:
```
agent.config.receivers.filelog.preserve_leading_whitespaces
agent.config.receivers.filelog.preserve_trailing_whitespaces
```

Let's not extend the values.yaml interface when it is not necessary

Reply:
That's right, I missed the part with that configuration. I'll close this PR